my_site/blog.php

```diff
@@ -0,0 +1,142 @@
+<!-- 
+ The blog for lab 10
+-->
+<?php
+    $current_page = 'todo';
+    $page_title = "To-Do List";
+    require 'common/required.php';
+?>
+<?php
+// MODAL for blod actual page
+session_start();
+
+if (!logged_in()) redirect('login.php?next=blog.php');
```

Comment:
This shouldn't exist. We want to be able to read the blog even if we're not logged in.